checklist/README.md

```diff
@@ -0,0 +1,71 @@
+# AMP Contribution Checklist Bot
+
+A GitHub app which posts checklists on pull requests.
+
+This app runs as a Google Cloud Functions deployment. It can be configured for deployment to any organization.
+
+## Interface
+
+The AMP Contribution Checklist Bot watches for all newly opened pull requests.
+
+If the pull request includes a new directory in `extensions/`, the bot will:
+
+1. edit the pull request's description to insert a checklist
+2. post a comment on the pull request metioning the added checklist
+
+## Webhooks
+
+The app subscribes to the following GitHub Webhooks:
+
+- [`PullRequestEvent`](https://developer.github.com/v3/activity/events/types/#pullrequestevent)
+  - `opened`: check the PR content and its comments to post checklist
```

Comment:
How do you want to handle draft PRs or PRs with "DO NOT SUBMIT" or similar things in the title? This is a common question we end up confronting after-the-fact with PR bots, worth answering here. If you want to ignore draft PRs and only comment once the PR is ready, you can filter on `payload.pull_request.draft` or something like that, and then you can listen for the `pull_requests.ready_for_review` event. The Owners Bot `index.js` has some logic like this you may find helpful.